tigerpath/majors_and_certificates/majors/GEO_2018.json

```
@@ -0,0 +1,467 @@
+{
+  "type": "Major",
+  "name": "Geosciences",
+  "code": "GEO",
+  "degree": "AB",
+  "year": 2018,
+  "urls": [
+    "https://ua.princeton.edu/academic-units/department-geosciences",
+    "https://geosciences.princeton.edu/undergraduate/programs-of-study"
+  ],
+  "contacts": [
+    {
+      "type": "Chair",
+      "name": "Bess B. Ward",
+      "email": "bbw@princeton.edu"
+    },
+    {
+      "type": "Associate Chair",
+      "name": "Thomas S. Duffy",
+      "email": "duffy@princeton.edu"
+    },
+    {
+      "type": "Director of Undergraduate Studies",
+      "name": "Adam C. Maloof",
+      "email": "maloof@princeton.edu"
+    }
+  ],
+  "req_list": [
+    {
+      "name": "Prerequisites",
+      "max_counted": 1,
+      "min_needed": "ALL",
+      "explanation": "Three prerequisite requirements for entering the concentration (MAT 104 (or AP), COS 126, and GEO 202 or 203). Students should elect GEO 202 or 203 with an eye toward their likely track of study. Students who need to take the other course for their specialization may count the course used as a prerequisite as an elective (in #5, below). With permission of the DUS, a student can substitute GEO 102/103, a Geosciences Freshman Seminar, or certain Geosciences 300/400 level courses for GEO202/203.",
+      "double_counting_allowed": true,
+      "req_list": [
+        {
+          "name": "Math",
+          "max_counted": 1,
+          "min_needed": 1,
+          "explanation": "Three prerequisite requirements for entering the concentration (MAT 104 (or AP), COS 126, and GEO 202 or 203).",
+          "course_list": [
+            "MAT 104"
+          ]
+        },
+        {
+          "name": "Computer Science",
+          "max_counted": 1,
+          "min_needed": 1,
+          "explanation": "Three prerequisite requirements for entering the concentration (MAT 104 (or AP), COS 126, and GEO 202 or 203).",
+          "course_list": [
+            "COS 126"
+          ]
+        },
+        {
+          "name": "Geoscience",
+          "max_counted": 1,
+          "min_needed": 1,
+          "explanation": "Three prerequisite requirements for entering the concentration (MAT 104 (or AP), COS 126, and GEO 202 or 203). Students should elect GEO 202 or 203 with an eye toward their likely track of study.",

```

Alternative. Based on [department website](https://geosciences.princeton.edu/undergraduate/programs-of-study).
Might be useful for students to keep in mind.
```suggestion
          "explanation": "Three prerequisite requirements for entering the concentration (MAT 104 (or AP), COS 126, and GEO 202 or 203). Students should elect GEO 202 or 203 with an eye toward their likely track of study.\n\nWith permission of the DUS, a student can substitute GEO 102/103, a Geosciences Freshman Seminar, or certain Geosciences 300/400 level courses for GEO 202/203.",
```